lib/usd/ui/layerEditor/layerTreeModel.cpp

```diff
@@ -300,13 +302,7 @@ void LayerTreeModel::rebuildModel(bool refreshLockState /*= false*/)
     _lastAskedAnonLayerNameSinceRebuild = 0;
 
     beginResetModel();
-
-    //  Note: do *not* call clear() here! Unfortunately, clear() itself calls,
-    //        beginResetModel() and endResetModel(). Qt does not detect the nested
-    //        begin/end/ So calling clear() would make the layer manager flicker
-    //        to be empty for a brief time.
-    if (rowCount() > 0)
-        removeRows(0, rowCount());
+    clear();
```

Comment:
Is this change required? Like the comments said, clear() would cause multiple notifications of changes to be sent and cause flickers. Any reason why changing it to call clear?